edb/edgeql/ast.py

```diff
@@ -154,12 +154,16 @@ class SortExpr(Base):
     nones_order: typing.Optional[NonesOrder] = None
 
 
-class AliasedExpr(Base):
+class Alias(Base):
```

Comment:
Do we want a more descriptive name for `Alias`?

Reply:
What about variable? With binding binds multiple "variables"?

It has to be something that encapsulated both "a variable that is an expression" (`AliasedExpr`) and "module scope" (`ModuleAliasDecl`).

Reply:
We can change this later.